LiveStream/Service/KsLive.swift

```diff
@@ -0,0 +1,39 @@
+import FirebaseAnalytics
+import FirebaseDatabase
+import FirebaseAuth
+
+public class KsLiveApp {
```

Comment:
do you want to rename this file to match the class name? `KsLive` -> `KsLiveApp`